tests/integration/benchmark/ir-llm/scripts/cw_metrics.py

```diff
@@ -0,0 +1,195 @@
+            if field_name in metrics:
+                if (
+                    field_name == "tokenizerFailed_Sum"
+                    or field_name == "emptyInferenceResponse_Sum"
+                    or field_name == "clientInvocationErrors_Sum"
+                ):
+                    if float(row["clientInvocations_Sum"]) > 0.0:
+                        field_value = (
+                            float(row[field_name])
+                            * 100
+                            // float(row["clientInvocations_Sum"])
+                        )
+                    else:
+                        field_value = 0.0
+
+                metric_data = [
+                    {
+                        "MetricName": metrics[field_name]["metric_name"],
+                        "Dimensions": [
+                            {
+                                "Name": "Model",
+                                "Value": model_name,
+                            },
+                            {
+                                "Name": "Endpoint",
+                                "Value": endpoint_type,
+                            },
+                            {
+                                "Name": "Container",
+                                "Value": container,
+                            },
+                            {
+                                "Name": "InstanceType",
+                                "Value": row["InstanceType"],
+                            },
+                            {
+                                "Name": "Concurrency",
+                                "Value": row["Concurrency"],
+                            },
+                        ],
+                        "Unit": metrics[field_name]["unit"],
+                        "Value": float(field_value),
+                    }
+                ]
+                response = cw.put_metric_data(
+                    Namespace=metrics_namespace, MetricData=metric_data
+                )
+                logging.info(
+                    "publish metric: %s, model: %s, endpoint: %s, container: %s, instance_type: %s, concurrency: %s, response: %s",
+                    metrics[field_name]["metric_name"],
+                    model_name,
+                    endpoint_type,
+                    container,
+                    row["InstanceType"],
+                    row["Concurrency"],
+                    response,
+                )
+
+
+def run_benchmark(config_yml, benchmark_config_dir, benchmark_metric_dir):
+    with open(config_yml, "r") as file:
+        config = yaml.safe_load(file)
+
+        if config is None:
+            logging.fatal("Invalid config.yml")
+        region = config.get("region", "us-west-2")
+        metrics_namespace = config.get("cloudwatch", {}).get(
+            "metrics_namespace", "Rubikon"
+        )
+        metrics = config.get("metrics", {})
+        hf_token = config.get("HF_TOKEN", "")
```

Comment:
Can we parse HF_TOKEN from env? I don't want us accidentally leaking credentials for hub access given that we routinely get added to private repos.

Maybe we could also explore a team CI HF account and persist a token as part of github secrets?